src/egon/data/datasets/scenario_parameters/parameters.py

```diff
@@ -105,6 +105,7 @@ def global_settings(scenario):
                 "coal": 20.2,  # [EUR/MWh]
                 "lignite": 4.0,  # [EUR/MWh]
                 "nuclear": 1.7,  # [EUR/MWh]
+                "biomass": 40,  # Dummyvalue, ToDo: Find a suitable source. To be fix in #268
```

Comment:
This is not needed anymore, right?

Reply:
Right. It was deleted in dd2488fe2470a489ba8f4fda8c5df79b03d4c410.